lib/services/elasticsearch.js

```diff
@@ -38,6 +38,60 @@ function ElasticSearch (kuzzle, options, config) {
     }
   });
 
+  this.errorMessagesMapping = [
+    {
+      // [illegal_argument_exception] object mapping [titi] can't be changed from nested to non-nested
+      regex: /^\[illegal_argument_exception\] object mapping \[(.*?)\] can't be changed from nested to non-nested$/,
+      replacement: 'Can not change mapping for field "$1" from nested to another type'
```

Comment:
Because one day I asked me if a type `object` is a nested or non-nested type and in fact no.